transcendence-app/src/chat/chat/infrastructure/db/chat-message.entity.ts

```diff
@@ -0,0 +1,17 @@
+export enum chatMessageKeys {
+  ID = '"id"',
+  SENDER_ID = '"senderId"',
+  RECIPIENT_ID = '"senderId"',
+  CONTENT = '"content"',
+  CREATED_AT = '"createdAt"',
+}
+
+export class ChatMessage {
+  constructor(
+    public id: string,
+    public senderId: string,
+    public recipientId: string,
+    public content: string,
+    public createdAt: Date,
+  ) {}
```

Comment:
suggestion: Please update the constructor to take an object as a parameter to instantiate it using the data from [pg.Result.rows](https://node-postgres.com/api/result#resultrows-arrayany).